nevergrad/functions/pownet/solver.py

```diff
@@ -0,0 +1,251 @@
+import os
+import importlib.util
+import requests
+from pyomo.opt import SolverFactory
+from pyomo.core import Var
+from operator import itemgetter
+from datetime import datetime
+import nevergrad as ng
+from nevergrad.functions.pyomo import core
+
+import git
+import pypownetr.model
+
+def download_dataset(dataset_name="cambodian", force_update=False):
+    #git clone <repository> <path>
+    dataset_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'pypownet')
```

Comment:
please use `form pathlib import Path`  and use the Path object everywhere instead of `os.path`, this will make the code way more readable